### PR TITLE
fix(electron security): Bump electron to 1.6.16

### PR DESCRIPTION
**Pull request by Empact**
To fix protocol handler vulnerability
https://electronjs.org/blog/protocol-handler-fix

### CODE REVIEW

**Comment by JimmyMow** [comment]
@Empact why not update the package.json?

**Comment by Empact** [comment]
Yeah, on that - I'm also switching to 1.6.16 in light of your latest change.

File: app/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   },
   "scripts": {
     "postinstall": "npm rebuild --runtime=electron --target=1.6.6 --disturl=https://atom.io/download/atom-shell --build-from-source",
```
Comment (JimmyMow) [inline app/package.json]: @Empact also thoughts on updating `--target` to the electron version we are packaging?

Comment (Empact) [reply app/package.json]: Yeah fixed in the current code.

**Comment by Empact** [comment]
@JimmyMow thoughts on moving back to the electron 1.7 series? What build command was failing here: 66c97eb3c41191d0980591ab170460019787214a

**Comment by JimmyMow** [comment]
@Empact we can move back to the `1.7` series. It was actually a `grpc` issue. `grpc@1.7.3` combined with electron was getting `404` on packaging for linux and was packaging incorrectly.

switching to `grpc@1.7.1` fixed this, I think we can go back to `1.7` series

**Comment by Empact** [comment]
Cool will do in a separate PR

**Comment by JimmyMow** [comment]
Yeah we can certainly improve our automated testing. Catching something like this would be tricky but an overall improvement is probably a priority todo at some point

**Pull request merged**